.github/workflows/performance.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     env:
       plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
       signin_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
-    runs-on: macos-12  # TODO: the legacy ObjC quickstarts don't run with Xcode 15.
+    runs-on: macos-14
```

Comment:
Thanks! So is this a solution or a workaround to the `Undefined symbol: __swift_FORCE_LOAD_$_swiftCompatibility56` error?

I'm still looking into it and want to double check that we should be trying to better understand why the builds broke.

Reply:
I suspect that it is a workaround based on https://stackoverflow.com/a/76604923 (`macos-12` wouldn't support Xcode 14.3). Might be interesting to try specifying Xcode 14.2, see if it fails, then specify 14.3 and see if it succeeds (though lower priority since we're only supporting Xcode 15.2+ now).